### PR TITLE
armadillo 4.650.4

### PR DESCRIPTION
**Pull request by zeeMonkeez**
Update armadillo to 4.650.4.
Tested on 10.10.3.

### CODE REVIEW

File: armadillo.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Armadillo < Formula
   homepage "http://arma.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/arma/armadillo-4.600.2.tar.gz"
-  sha1 "1b008706f52b154faeec3f6fcc8c98076ae9e5b9"
+  url "https://downloads.sourceforge.net/project/arma/armadillo-4.650.4.tar.gz"
+  sha1 "66525126e5b7d44d5f0b1689052e29ff662824a6"
```
Comment (dpo) [inline armadillo.rb]: Could you please use the `sha256`? You can find it with `shasum -a 256 /Library/Caches/Homebrew/armadillo-4-650.4.tar.gz`.

File: armadillo.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 class Armadillo < Formula
   homepage "http://arma.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/arma/armadillo-4.600.2.tar.gz"
-  sha1 "1b008706f52b154faeec3f6fcc8c98076ae9e5b9"
+  url "https://downloads.sourceforge.net/project/arma/armadillo-4.650.4.tar.gz"
+  sha1 "ccc56580ec932ec9beade5679b07ecbed889e9b602866f673d1a05a7692b4bf8"
```
Comment (iml) [inline armadillo.rb]: `sha256`

Comment (zeeMonkeez) [inline armadillo.rb]: sorry. don't know where my head is.

**Comment by dunn** [comment]
Is there a reason not to bump up to 5.000?

**Comment by zeeMonkeez** [comment]
5.000.0 had not been released when this PR was created ... Should I create a new PR for this?

**Comment by dunn** [comment]
Oh, sorry, I thought it had.

Sure!  Or I can when I get the chance.

**Pull request closed without merging**